CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
 
 -   Added multi voting key file support.
 -   Added `updateVotingKeys` command.
+-   The `bootstrap` preset is not the default anymore. The name must be provided via --preset or as a custom preset field. 
+-   Added custom preset to the target folder. It's not required when upgrading the node without a configuration change.
```

Comment:
It's **not** required? What do you mean?

Reply:
Soo, if you want to change a custom preset configuration, (like the friendly name), you need to provide the custom preset with --upgrade. If you are just upgrading the configuration, the custom preset is not required anymore. 

"The --customPreset param is only required when there is a configuration change, like a new 'friendlyName'"

Reply:
Ah, OK, understood. Then yeah, something like:

```suggestion
-   When upgrading, the --customPreset param is now only required if there is a configuration change, since
    the properties from the original custom preset are fully cached in the target folder.
```

But this is only the CHANGELOG so it's no big deal :D